pom.xml

```diff
@@ -47,17 +36,21 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.1</version>
                 <configuration>
-                    <argLine>
-                        -javaagent:"${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar"
-                    </argLine>
+                    <systemProperties>
+                        <property>
+                            <name>allure.results.directory</name>
```

Comment:
It is much better to keep results directory configured in `allure.properties` file, because IDE's like to ignore configuration specified in pom.xml.

See https://github.com/allure-examples/allure-junit5-example/blob/master/src/test/resources/allure.properties